eoscostarica/gGoods
# fix: integration errors

### Fix integration errors

### What does this PR do?

- Resolve #N/A

### Steps to test

1. make run
2. navigate to http://localhost:9695
3. check the available actions 

#### CheckList
